qutebrowser/qutebrowser.py

```diff
@@ -115,23 +115,10 @@ def get_argparser():
     # For the Qt args, we use store_const with const=True rather than
     # store_true because we want the default to be None, to make
     # utils.qt:get_args easier.
```

Comment:
I didn't realize this earlier, but this comment is now obsolete, so you can get rid of it too.